hypha/apply/projects/models/report.py

```diff
@@ -12,7 +12,12 @@
 from django.utils import timezone
 from django.utils.functional import cached_property
 from django.utils.translation import gettext_lazy as _
+from wagtail.fields import StreamField
 
+from hypha.apply.funds.models.mixins import AccessFormData
+from hypha.apply.projects.blocks import ProjectApprovalFormCustomFormFieldsBlock
```

Comment:
The name is specific about it being used for the approval form. Someone could change it believing it does what the name applies.

Better to create a new class for this use case. It can be identical but with a name explaining what is does.

Reply:
Similar to above, maybe we can extract the common code into a truly common/re-usable class?

Reply:
I renamed this class to the more generic thing that it is and added a comment. Does this help resolve?